backend/protocol_rpc/endpoints.py

```diff
@@ -504,7 +507,187 @@ def get_transaction_count(
 def get_transaction_by_hash(
     transactions_processor: TransactionsProcessor, transaction_hash: str
 ) -> dict | None:
-    return transactions_processor.get_transaction_by_hash(transaction_hash)
+    def vote_name_to_number(vote_name: str) -> int:
```

Comment:
@kstroobants we may want to use these functions, why are they scoped just inside this function?